node/src/components/consensus/protocols/highway/config.rs

```diff
@@ -20,6 +20,7 @@ pub struct Config {
     /// state from peers.
     pub standstill_timeout: TimeDiff,
     /// If another `standstill_timeout` passes assume we failed to join the network and restart.
+    #[serde(default = "default_shutdown_on_standstill")]
```

Comment:
The default for `bool` is `false`, so just using `#[serde(default)]` would be equivalent. It might be slightly less readable, though.

Reply:
Exactly, it's not entirely obvious, so I went with a function. If you prefer I can change it.

Reply:
I don't have a strong preference, it's good enough to me :+1: